tiatoolbox/utils/env_detection.py

```diff
@@ -344,12 +340,12 @@ def check_pixman_using_macports(versions: list) -> tuple[list, str]:
         flags=re.MULTILINE,
     )
     if matches:
-        versions = [version_to_tuple(matches.group(1))]
+        versions = [version_to_tuple(match) for match in matches]
```

Comment:
`re.finall()` returns a list of strings, not a `match` object, so it doesn't have the  `group` attribute.